plugins/kube-ps1/README.md

```diff
@@ -0,0 +1,76 @@
+Kubernetes prompt for zsh
+=========================
+
+A Kubernetes (k8s) zsh prompt that displays the current cluster cluster
+and the namespace.
+
+Inspired by several tools used to simplify usage of kubectl
+
+NOTE: If you are not using zsh, check out [kube-ps1](https://github.com/jonmosco/kube-ps1) designed for bash
+as well as zsh.
+
+## Requirements
+
+The default prompt assumes you have the kubectl command line utility installed.  It
+can be obtained here:
+
+[Install and Set up kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+If using this with OpenShift, the oc tool needs installed.  It can be obtained from here:
+
+[OC Client Tools](https://www.openshift.org/download.html)
+
+## Helper utilities
+
+There are several great tools that make using kubectl very enjoyable.
+
+[kubectx and kubenx](https://github.com/ahmetb/kubectx) are great for
+fast switching between clusters and namespaces.
+
+## Prompt Structure
+
+The prompt layout is:
+
+```
+(<logo>|<cluster>:<namespace>)
+```
+
+Supported platforms:
+* k8s - Kubernetes
+* ocp - OpenShift
+
+## Install
+
+1. Clone this repository
+2. Source the kube-ps1.zsh in your ~./.zshrc
+
+ZSH:
+```
+source path/kube-ps1.sh
```

Comment:
A suggestion, in order to be sourced at the startup of a new terminal by [oh-my-zsh.sh](https://github.com/robbyrussell/oh-my-zsh/blob/master/oh-my-zsh.sh) script, plugin names are added to a variable in zsh.rc called `plugins`, as explained in the [wiki](https://github.com/robbyrussell/oh-my-zsh/wiki/Plugins#example). For this, you need to rename the file to `kube-ps1.plugin.zsh`. It would be more practical for using the plugin IMHO.

Reply:
I'd like to use this plugin, but it would mess up my PROMPT, which I'm quite content with it. However, I'd love to use this plugin as an addition to my current PROMPT, that would be awesome! @jonmosco would you consider that? I'd write a PR when I find time.

Reply:
I can rename that, no problem.

This can be used in addition to your current PROMPT and should not mess your current one up.  I will add the new version of this along with the rename as well.  Also, check https://github.com/jonmosco/kube-ps1 for other instructions related to this prompt.